CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ cmake_minimum_required(VERSION 2.6)
 add_definitions(-D UNIX)
 add_definitions(-D LINUX)
 add_definitions(-D _GNU_SOURCE)
+add_definitions(-std=c++11 -fpermissive)
```

Comment:
Using -fpermissive is not particularly nice. This commit seems to indicate that supporting C++11 requires some code changes besides changed compiler flags. Thus it would be much nicer too fix all compile errors instead of disabling them.